Client/Frontend/Home/HomepageViewController.swift

```diff
@@ -312,7 +297,7 @@ class HomepageViewController: UIViewController, HomePanel, GleanPlumbMessageMana
     // MARK: - Contextual hint
     private func prepareJumpBackInContextualHint(onView headerView: LabelButtonHeaderView) {
         guard contextualHintViewController.shouldPresentHint(),
-              !shouldDisplayHomeTabBanner
+              !viewModel.shouldDisplayHomeTabBanner
```

Comment:
Wanted to double check here - the ticket says HomeTabBanner & JumpBackIn CFR can show up at the same time. This is okay to keep?

Reply:
I will double check with product because the ticket does say to show at the same time but every time we talk about it the idea was to keep the logic just to make sure the CFR was showing again

Reply:
Yeah last I remember, we didn't want them both showing together because the anchor point could be off screen on smaller devices.

Reply:
Check with product and we want to keep this logic to avoid showing the CFR while the card is presented